tsl/test/expected/cagg_on_cagg.out

```diff
@@ -1084,16 +1134,39 @@ SELECT
 FROM :CAGG_NAME_1ST_LEVEL
 GROUP BY 1
 WITH NO DATA;
-psql:include/cagg_on_cagg_validations.sql:43: ERROR:  cannot create continuous aggregate with incompatible bucket width
+psql:include/cagg_on_cagg_validations.sql:44: ERROR:  cannot create continuous aggregate with incompatible bucket width
 DETAIL:  Time bucket width of "public.conditions_summary_2" [2] should be greater or equal than the time bucket width of "public.conditions_summary_1" [4].
 \d+ :CAGG_NAME_2TH_LEVEL
+-- Check for incorrect CAGGs
+\if :INTERVAL_TEST
+  INSERT INTO conditions ("time", temperature, device_id) VALUES ('2022-01-01 00:00:00-00', 10, 4);
+  INSERT INTO conditions ("time", temperature, device_id) VALUES ('2022-01-15 01:00:00-00', 20, 4);
+  INSERT INTO conditions ("time", temperature, device_id) VALUES ('2022-01-31 01:00:00-00', 30, 4);
+  CALL refresh_continuous_aggregate(:'CAGG_NAME_1ST_LEVEL', NULL, NULL);
+  CALL refresh_continuous_aggregate(:'CAGG_NAME_2TH_LEVEL', NULL, NULL);
+  CREATE MATERIALIZED VIEW :CAGG_NAME_3TH_LEVEL
+  WITH (timescaledb.continuous) AS
+  SELECT
+    \if :IS_TIME_DIMENSION_WITH_TIMEZONE_2TH
+      time_bucket(:BUCKET_WIDTH_3TH, "bucket", :'BUCKET_TZNAME_2TH') AS bucket
+    \else
+      time_bucket(:BUCKET_WIDTH_3TH, "bucket") AS bucket
+    \endif
+  FROM :CAGG_NAME_2TH_LEVEL
+  GROUP BY 1
+  WITH DATA;
+  \d+ :CAGG_NAME_3TH_LEVEL
+  SELECT * from :CAGG_NAME_3TH_LEVEL;
+  DROP MATERIALIZED VIEW IF EXISTS :CAGG_NAME_3TH_LEVEL;
+  DELETE FROM conditions WHERE device_id = 4;
+\endif
```

Comment:
I guess this new code block should be moved to after the `\set VERBOSITY terse` below, no?